tests/unit/expression_test.py

```diff
@@ -1064,6 +1298,32 @@ def test_attribute_z3expr_error() -> None:
         First(Call("X")).z3expr()
 
 
+@pytest.mark.parametrize(
+    "attribute, tac_name",
+    [
+        (Size("X"), "X'Size"),
```

Comment:
Will there be a separate construct for attributes in our TAC in the future (e.g. `R := Attribute(x, Size)`)?

Reply:
At the moment, variables (`BoolVar` or `IntVar`) are used to represent attributes. For the verification, this should be sufficient. But I can imagine that this doesn't work well for code generation. If that is the case, introducing a separate construct would make sense. I think we will find that out when trying to adapt the code generator to the TAC.

Reply:
I see. Thanks!